tests/integration/test_ask_tell_optimization.py

```diff
@@ -46,9 +46,7 @@
 from trieste.space import Box, SearchSpace
 from trieste.types import State, TensorType
 
-
-@random_seed
-@pytest.mark.parametrize(
+BRANIN_OPTIMIZER_PARAMS = (
```

Comment:
How about more neutral name, e.g. `TEST_SCENARIOS` or simply `OPTIMIZER_PARAMS`? They are not super specific to branin, and we use them as inputs to both branin and quadratic tests